core/server/data/schema/schema.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
     posts: {
-        id: {type: 'increments', nullable: false, primary: true},
+        id: {type: 'string', nullable: false, primary: true},
```

Comment:
The combination of changes in this commit mean that all the foreign key fields are getting 191 chars, surely they only need 24? Is there a reason for leaving them longer than needed?